commons/phone/phone_contact_list.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactList",
+  "doc": "Data describing the number of contacts in the the list of contacts and its change since the previous check.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "contactsAdded", "type": ["null", "int"], "doc": "Number of new contacts added to the list of contacts since the previous measurement that was sent. The value is null if this is the first measurement or if the previous measurement is unknown to the sender."},
+    {"name": "contactsRemoved", "type": ["null", "int"], "doc": "Number of new contacts removed from the list of contacts since the previous measurement that was sent. The value is null if this is the first measurement or if the previous measurement is unknown to the sender."},
```

Comment:
Could you please add `"default": null` to `contactsAdded` and `contactsRemoved`?

Reply:
Done